src/ui/components/accounts/activeAccountCard.tsx

```diff
@@ -42,7 +40,13 @@ export function ActiveAccountCard({
             {account.name}
           </div>
 
-          <Balance balance={balance} isShortFormat={false} showAsset split />
+          {amountInUsd !== null ? (
+            <p className={styles.accountAmount}>{`$${
+              Math.ceil(+amountInUsd * 100) / 100
```

Comment:
It's better to use `BigNumber` class for things like this to preserve precision. And so `amountInUsd` prop should accept `BigNumber | null`. It has `toFixed` and `toFormat` methods.